web/Controllers/PropertyController.cs

```diff
@@ -123,4 +124,21 @@ public ActionResult AddNewProperty_Cancel()
         // Go to View Properties page
         return RedirectToAction("ViewProperties", "Landlord");
     }
-}
+
+    [HttpGet]
+    [Route("/property/view")]

```

Comment:
It'd be nice if the property ID was in the route e.g. `property/32/view`

Reply:
ok done